### PR TITLE
Add London F2F minutes

### PR DESCRIPTION
**Pull request by tobie**
~Still pending approval of guests for additional notes.~

### CODE REVIEW

File: meetings/2019-03-f2f.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+### Scope and Role of the AC
+
+* AC is Advisory to the TSC.
+* Advice is non-binding.
+* AC can provide opinions / advice / findings.
+* Once we have a Foundation, we will have a board
+  * AC should be represented on the Board
+  * TSC should be represented on the Board
+  * The Board will be respnsible for finance, IP, and governance
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The Board will be responsible for finance, IP, and governance

File: meetings/2019-03-f2f.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+* Tobie Langel @tobie
+* Charles Vazac @cvazac
+* Joe Alicata @wirelessjoe
+* Levi Durfee @levidurfee
+* Léonie Watson @ljwatson
+
+### What is the TSC
+
+* Technical Steering Committee.
+* Inspired by NodeJS style terminology.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: It is inspired by NodeJS style terminology.

File: meetings/2019-03-f2f.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+* Responsible for handling project day to day activity.
+* Roadmap, direction. Project management / tech lead distributed throughout a group.
+* TSC delegate to working groups.
+* 3 out of 7 members are from Google for now.
+* Goal is that no more than 1/3 of members are from the same company.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The goal is that no more than 1/3 of members are from the same company.

File: meetings/2019-03-f2f.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+* AMP Vision - https://www.ampproject.org/about/mission/
+  * Right intent but too vague
+* [issue meta#9][meta9]
+  * No - AMP AC needs "critical friends"
+  * We think that the vision / mission needs works
+  * Vision: A strong, user-first open web forever.
+  * **AI(@edent, @ljwatson): draft an AC opinion on the topic.**
+  * Once that opinion is drafted and published by the AC, and depending on the resolution of the TSC, the AC will revisit [issue meta#9][meta9]. The current sentiment is that those restrictions shouldn't apply as long as the AC's membership is properly balanced.
+* How do we get more members and more involvement from existing members?
+* Target size of AC is 6 - 12 member.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The target size of AC is 6 - 12 member.

File: meetings/2019-03-f2f.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+* Current size is 16.
+* No fixed size.
+* AC must be representative of different stakeholders
+* Ideally, we'd love for AC members to be elected? In practice, that's impossible.
+* We've received requests from people to join
+* Membership length is 1 year.
+  * Members in good standing will be asked to rejoin.
+  * Members are under no obligation to rejoin
+  * Year starts on Jan, 1st.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The year starts on Jan, 1st.

File: meetings/2019-03-f2f.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+  * Members can resign at any time
+  * We need a concept of "Good Standing" (see for example [W3C's definition](https://www.w3.org/2005/10/Process-20051014/process.html#good-standing)), still TBD.
+* **AI(@tobie): open a pull request against the AC's work mode document to reflect agreed upon term durations.**
+* Current AC members approve any new AC member
+  * Consensus based process
+  * AC want to ensure a diverse group of representative (diversity of location, thought, industry, gender, disability, race, etc)
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: AC wants to ensure a diverse group of representative (diversity of location, thought, industry, gender, disability, race, etc)

File: meetings/2019-03-f2f.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+  * "It is the AC's duty to ensure the AC is as representative and diverse as possible. When AC membership places are available, AC members commit to proactively reaching out to groups who are not currently represented. AC members will use Single Transferable Vote to select new members in a secret ballot."
+  * **AI(@tobie): open a pull request against the AC's workmode document to include a description of how new AC members are elected.**
+
+### Is AMP a product or an Open Source Project
+
+* Currently the perception is that it is a Google Product.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Currently, the perception is that it is a Google Product.

File: meetings/2019-03-f2f.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+* Only Google employees can merge requests for now (plan is for this to move to <33%)
+* Majority or changes are by Google employees
+* AMP Cache is not open source. (Reason: it's too tied to infrastructure to make sense open sourcing it. It couldn't be re-used.)
+* The perception is that AMP is beneficial to Google - not for the open web.
+* AMP needs to be separated - and seen to be separated - from Google. Too much interplay between Search, Browser, and AMP.
+* AC may advise both the TSC (open source) and the wider community (products).
+* AMP must share their user research openly.
+* User research data must be available to the AC/TSC to allow them to make decisions.
+* **AI(@edent, @ljwatson): draft a paper on results of user research.**
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: **AI(@edent, @ljwatson): draft a paper on the results of user research.**

File: meetings/2019-03-f2f.md [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+* URL Problem
+  * Dodgy sites get to use the google.com URL
+     * Users find this confusing
+     * It is seen as breaking the web
+     * This is an unfortunate byproduct of early design decisions
+  * WebPackaging being the answer?
+     * Not a standard (yet)
+     * Not supported in all browsers (Firefox, Safari)
+  * Given that the URL problem is a user-unfriendly pattern, and that two major browsers are unlikely to support WebPackaging, the AC needs to provide advice to the TSC on fixing this.
+* Issue with pref search treatment
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The issue with pref search treatment

File: meetings/2019-03-f2f.md [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+  * How do we account for web pages "cheating" on performance test? Dieselgate style.
+  * You have to use AMP to be in Google News Carousel. Discovery experience. This is perceived as discriminatory.
+  * Developers don't want multiple pipelines for their content. They have been incentivised by being at the top of search results.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Developers don't want multiple pipelines for their content. They have been incentivized by being at the top of search results.

File: meetings/2019-03-f2f.md [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+* Terence Eden @edent
+* Tobie Langel @tobie
+* Charles Vazac @cvazac
+* Joe Alicata @wirelessjoe
+* Léonie Watson @ljwatson
+
+#### Agenda:
+
+* ad-tech primer -> define what the AC's role would be in that space
+* organise verticals in groups (AC/TSC dependant?) [issue #5][5] [issue #15][15]
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: organize verticals in groups (AC/TSC dependant?) [issue #5][5] [issue #15][15]

File: meetings/2019-03-f2f.md [hunk 12]
```diff
@@ -0,0 +1,247 @@
...
+* foundation update
+* meeting around outreach w/ Ben Morss
+* accessibility + [issue #5][5]
+* AMP for email
+* Document AMP architecture + update standard list
+
+### Foundation
+
+* Tobie gives overview
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Tobie gives an overview

File: meetings/2019-03-f2f.md [hunk 13]
```diff
@@ -0,0 +1,247 @@
...
+* How are they going to be contacted? Outreach process.
+  * Outreach could be done by the foundation itself
+  * Ask AC/TSC membership would they be interested in joining the foundation
+  * At what price point?
+  * Whether we want the foundation limited to just legal/accounting or include outreach/etc.
+* Foundation should act as a steward for content when / if AMP caches are turned off.
+
+### Ad-tech primer / revenue
+
+* Joe gives overview
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Tobie gives an overview

Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Joe gives an overview

File: meetings/2019-03-f2f.md [hunk 14]
```diff
@@ -0,0 +1,247 @@
...
+* containerized in AMP (sandbox)
+* system:
+  * good doc on the topic in the AMP doc
+  * container in the page
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: the container in the page

File: meetings/2019-03-f2f.md [hunk 15]
```diff
@@ -0,0 +1,247 @@
...
+  * add server
+  * inside the code that comes back from the ad-server
+* As a result: limitations
+  * creative
+  * data (lower access to targeting data)
+* "Auction"-based ad-style weren't supported at first
+* https://www.ampproject.org/docs/ads/introduction_ads
+* Add-network's role: supply/demand aggregator
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Should this be Ad-network's?

File: meetings/2019-03-f2f.md [hunk 16]
```diff
@@ -0,0 +1,247 @@
...
+* benefit from a user's perspective: speed
+* AMP HTML Ads => AMP in the ad itself, served on any kind of Web page
+* Who guarantees the AMP format validity in that case?
+
+### Accessibility
+
+See [issue #5][5].
+
+* Many of the components are not sufficiently accessible.
+* Accessibility tree - usually used by browsers - they understand what ever HTML element does.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Accessibility tree - usually used by browsers - they understand whatever HTML element does.

File: meetings/2019-03-f2f.md [hunk 17]
```diff
@@ -0,0 +1,247 @@
...
+  * AMP doesn't rely on native elements enough
+* Media Player is a real concern. Controls show up as list boxes rather than buttons
+* Same kinds of issues for the carousel
+* Some bits are good - like collapsing
+* Is there a process for accessibility checking before a component is added?
+  * No - but there needs to be
+  * The AC would like to see a process in place by 2020-01-01
+* The AC would like all components to meet WCAG 2.1 before release
+* By the AMP Contributor Summit - produce a roadmap and timetable for raising the existing components up to the expected standard
+* **AI(): reach out to TSC/Accessibility WG and ask for audit of existing components.**
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: **AI(): reach out to TSC/Accessibility WG and ask for an audit of existing components.**

File: meetings/2019-03-f2f.md [hunk 18]
```diff
@@ -0,0 +1,247 @@
...
+* AMP documentation should specifically show how to make elements accessible
+* **AI(): reach out to TSC/Documentation WG to ask for related improvements to documentation.**
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: **AI(): reach out to TSC/Documentation WG to ask for related improvements to the documentation.**

File: meetings/2019-03-f2f.md [hunk 19]
```diff
@@ -0,0 +1,247 @@
...
+* AMP Validator should fail pages which do not pass automated a11y tests (100% score on lighthouse?)
+  * Byproduct - AMP Caches should not serve pages which do not validate against a11y tools
+* Offer our help to the WG
+* **AI(@tobie): organize meeting with a11y WG. Either invite them to AC meeting or join one of their calls. **
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: **AI(@tobie): organize a meeting with a11y WG. Either invite them to AC meeting or join one of their calls. **

File: meetings/2019-03-f2f.md [hunk 20]
```diff
@@ -0,0 +1,247 @@
...
+* Add horizontal review requirements in the existing process.
+* **AI(): author review requirements for a11y. Share them with the TSC/a11y WG.**
+
+* We don't have a clear measure of privacy
+* what are the privacy/security requirements to ship a new feature?
+* **AI(): reach out to TSC to ask for list of privacy/security requirements required to ship a new feature.**
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: **AI(): reach out to TSC to ask for a list of privacy/security requirements required to ship a new feature.**

File: meetings/2019-03-f2f.md [hunk 21]
```diff
@@ -0,0 +1,247 @@
...
+* Avoid fingerprinting as a target?
+
+### Outreach
+
+* Ben Morss joins the the meeting over VC
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Ben Morss joins the meeting over VC

File: meetings/2019-03-f2f.md [hunk 22]
```diff
@@ -0,0 +1,247 @@
...
+* Would love the help of more companies evangelizing for AMP
+* Evangelising is essentially about explaining AMP in terms of a Web-component library
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Evangelizing is essentially about explaining AMP in terms of a Web-component library

File: meetings/2019-03-f2f.md [hunk 23]
```diff
@@ -0,0 +1,247 @@
...
+* DevRel direction inside of Google?
+  * Small extension
+* Google Sales are incentivising people to use AMP
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Google Sales are incentivizing people to use AMP

File: meetings/2019-03-f2f.md [hunk 24]
```diff
@@ -0,0 +1,247 @@
...
+* Too few people writing documentation
+* Desire to make it accessible
+* wrt accessibility, everyone thinks it's someone else's job
+
+### Organise verticals in groups
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Organize verticals in groups

File: meetings/2019-03-f2f.md [hunk 25]
```diff
@@ -0,0 +1,247 @@
...
+
+See [issue #5][5] and [issue #15][15].
+
+* Should these vertical groups be AC or TSC dependant?
+  * Vertical groups could be AC, while horizontal review is clearly TSC.
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Vertical groups could be AC, while the horizontal review is clearly TSC.

File: meetings/2019-03-f2f.md [hunk 26]
```diff
@@ -0,0 +1,247 @@
...
+* Goal is to be more inclusive
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: The goal is to be more inclusive

File: meetings/2019-03-f2f.md [hunk 27]
```diff
@@ -0,0 +1,247 @@
...
+* The AC suggests W3C-inspired "interest groups."
+  * Open membership
+  * Conversations are public
+  * How do we create a group?
+  * Non-binding advisory role
+  * Similar creation requirements as working groups
+  * AC responsible? TBD
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Is the AC responsible? TBD

File: meetings/2019-03-f2f.md [hunk 28]
```diff
@@ -0,0 +1,247 @@
...
+* **AI(@tobie): update the relevant issues with the AC's proposal.**
+
+### AMP for email
+
+* Outcomes?
+* Formal user needs? User research?
+* User needs cannot be well understood by developers without user research
+* Publish the research (suitably anonymised)
```
Comment (levidurfee) [inline meetings/2019-03-f2f.md]: Publish the research (suitably anonymized)

File: meetings/2019-03-f2f.md (hunk 1 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]:
```suggestion
  * The Board will be responsible for finance, IP, and governance
```

File: meetings/2019-03-f2f.md (hunk 2 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]: OK, I'll only fix spelling errors (without getting in the British vs. US english debate). They're notes, so I'm fine with quasi sentences and the like.

File: meetings/2019-03-f2f.md (hunk 15 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]:
```suggestion
* Ad-network's role: supply/demand aggregator 
```

File: meetings/2019-03-f2f.md (hunk 7 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]:
```suggestion
* Currently, the perception is that it is a Google Product.
```

File: meetings/2019-03-f2f.md (hunk 16 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]:
```suggestion
* Accessibility tree - usually used by browsers - they understand whatever HTML element does.
```

File: meetings/2019-03-f2f.md (hunk 21 above)
Comment (tobie) [reply meetings/2019-03-f2f.md]:
```suggestion
* Ben Morss joins the meeting over VC
```

File: meetings/2019-03-f2f.md [hunk 29]
```diff
@@ -0,0 +1,296 @@
...
+* Some clearly voiced concerns that the AC doesn't actually have any teeth and cannot impact the project in ways that would run counter to Google's commercial interests?
+* IETF unconvinced that Google is fully committed to the WebPackaging standard.
+
+[19573]: https://github.com/ampproject/amphtml/issues/19573
+[neverslow]: https://news.ycombinator.com/item?id=19087717
+[featurepolicy]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy
+[miplaunch]: https://www.dragonmetrics.com/baidu-launches-their-own-amp-mobile-instant-pages-mip/
+[mipwebsite]: https://www.mipengine.org/
+
+***
```
Comment (tobie) [inline meetings/2019-03-f2f.md]:
```suggestion
```

**Pull request merged**